portal-impl/src/com/liferay/portlet/asset/util/AssetSearcher.java

```diff
@@ -639,6 +648,17 @@ protected void postProcessFullQuery(
 		return ArrayUtil.toArray(filteredCategoryIds.toArray(new Long[0]));
 	}
 
+	private static void _escapeSpecialChars(String[] keywords){
```

Comment:
this function should return String[], we could not aware of parameters modified by void function.